### PR TITLE
[ODS-4309] Integrate any remaining TPDM model changes 

### PR DESCRIPTION
**Pull request by tnunnerydl**

### CODE REVIEW

File: Application/EdFi.Ods.Extensions.TPDM/Artifacts/MsSql/Structure/Ods/0040-EXTENSION-TPDM-IdColumnUniqueIndexes.sql [hunk 1]
```diff
@@ -1,8 +1,3 @@
--- SPDX-License-Identifier: Apache-2.0
```
Comment (tbmeador-dlp) [inline Application/EdFi.Ods.Extensions.TPDM/Artifacts/MsSql/Structure/Ods/0040-EXTENSION-TPDM-IdColumnUniqueIndexes.sql:1]: we need to keep these license headers

Comment (tbmeador-dlp) [reply Application/EdFi.Ods.Extensions.TPDM/Artifacts/MsSql/Structure/Ods/0040-EXTENSION-TPDM-IdColumnUniqueIndexes.sql:1]: at least according to the docs:
https://techdocs.ed-fi.org/display/EDFIODS/License+and+Copyright+Notifications
https://techdocs.ed-fi.org/display/EDFIODS/Analysis+Spike%3A+Tool+Support+for+File+Headers

Comment (tnunnerydl) [reply Application/EdFi.Ods.Extensions.TPDM/Artifacts/MsSql/Structure/Ods/0040-EXTENSION-TPDM-IdColumnUniqueIndexes.sql]: license headers are added

File: Extensions/TPDM/TPDMMedataEd/package.json [hunk 2]
```diff
@@ -1,3 +1,7 @@
+// SPDX-License-Identifier: Apache-2.0
+//  Licensed to the Ed-Fi Alliance under one or more agreements.
+//  The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+//  See the LICENSE and NOTICES files in the project root for more information.
```
[the remark this answers is not in the document]
Comment (tnunnerydl) [reply Extensions/TPDM/TPDMMedataEd/package.json]: Removed.

File: Application/EdFi.Ods.Extensions.TPDM/Artifacts/Metadata/ApiModel-EXTENSION.json [hunk 3]
```diff
@@ -3,7 +3,6 @@
 //  The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 //  See the LICENSE and NOTICES files in the project root for more information.
```
[the remark this answers is not in the document]
Comment (tnunnerydl) [reply Application/EdFi.Ods.Extensions.TPDM/Artifacts/Metadata/ApiModel-EXTENSION.json]: Removed

**Pull request merged**